server/config.js

```diff
@@ -19,32 +36,13 @@ module.exports = {
     },
 
     main: {
-        connection: process.env.NODE_ENV === 'test' ? 'disk' : 'mysql',
+        connection: 'schwifty',
         secrets: {
             jwtSecret: Creds.secrets.jwtSecret
         }
     },
 
-    dogwater: {
-        connections: {
-            mysql: {
-                adapter: 'mysql',
-
-                host      : 'localhost',
-                port      : 3306,
-                user      : Creds.mysqlCreds.user,
-                password  : Creds.mysqlCreds.password,
-                database  : 'user-boilerplate',
-                connectionLimit: 30,
-                // Optional
-                charset   : 'utf8',
-                collation : 'utf8_swedish_ci'
-            },
-            disk: { adapter: 'disk' }
-        },
-        adapters: {
-            disk: SailsDisk,
-            mysql:  SailsMySQL
-        }
+    schwifty: {
+        knex: process.env.NODE_ENV === 'test' ? knexTest : knexLive
```

Comment:
I just poked around in the knex code and it doesn't seem like it's set up to handle anything like this (though it'd be cool if it did).